src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -377,7 +377,8 @@ public function getContainer()
      * @param string $name      The cache name prefix
      * @param string $extension File extension of the resulting file
      *
-     * @deprecated since version 3.3, to be removed in 4.0.
+     * @deprecated since version 3.3, to be removed in 4.0. Call to this can be completely removed if using composer and PHP 7
+     * @see https://symfony.com/blog/new-in-symfony-3-3-deprecated-the-classloader-component
```

Comment:
The link to the blog post should be removed, we don't add such references in the code base.
Not sure that docblock message needs to be changed, but the same message below should be (L385). 
About the text, I'd propose:
`'The "'.__METHOD__.'()" method is deprecated since version 3.3 and will be removed in 4.0 as the class cache is not needed anymore when using PHP 7.0.'`

Reply:
@delboy1978uk don't forget about this comment :)